src/tls/openssl.c

```diff
@@ -319,7 +403,9 @@ static int load_system_certificates(struct tls_context *ctx)
     /* For Windows use specific API to read the certs store */
 #ifdef _MSC_VER
     return windows_load_system_certificates(ctx);
-#endif
+#elif defined(__APPLE__)
+    return macos_load_system_certificates(ctx);
```

Comment:
Basically, it would be good to support security framework on macOS. But once we support it on macOS, we need to describe how it works and how to add certificates in the root store of security framework.